src/cpp/client/cronet_credentials.cc

```diff
@@ -1,64 +0,0 @@
-/*
- *
- * Copyright 2016 gRPC authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- */
-
-#include <grpcpp/security/credentials.h>
-
-#include <grpc/grpc_cronet.h>
-#include <grpcpp/channel.h>
-#include <grpcpp/support/channel_arguments.h>
-#include "src/cpp/client/create_channel_internal.h"
-
-namespace grpc {
-
-class CronetChannelCredentialsImpl final : public ChannelCredentials {
```

Comment:
Why was this removed?

Reply:
It's removed because it belongs to the main BUILD file which cannot have cronet-related rules. It can have a separate BUILD file like Objective-C does. Even though it's removed in this CL, it's possible to revive it with the separate BUILD file.

Reply:
Can you please add it back somewhere? I have some work in progress (Cronet C++ tests) that depends on the existence of this code.

Reply:
Also need to ensure that cronet_credentials.cc is included in gRPC-C++.podspec as iOS tests will be using cocoapods to build Cronet C++

Reply:
This file is back again but not included in BUILD so please make your own rule in the separate BUILD file.

Reply:
Thanks! Please add a header that defines `CronetChannelCredentials` as it has been removed from credentials.h. Also, please verify that cronet_credentials.cc and the new header that includes `CronetChannelCredentials` are available in `gRPC-C++.podspec`.

Reply:
Got it back again.

Reply:
New header file for cronet is added but it'd be better to integrate with podspec later since it should be refactored somehow not to be included in the main BUILD file.

Reply:
gRPC-C++.podspec now has cronet files.